### PR TITLE
Create Azure Component template for bindings.mysql

### PR DESCRIPTION
**Pull request by MregXN**
# Description
Create Azure Component template for bindings.mysql

### CODE REVIEW

File: bindings/mysql/metadata.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - name: exec
+      description: "The exec operation can be used for DDL operations (like table creation), as well as INSERT, UPDATE, DELETE operations which return only metadata (e.g. number of affected rows)."
+    - name: query
+      description: "The query operation is used for SELECT statements, which returns the metadata along with data in a form of an array of row values."
+    - name: close
+      description: "The close operation can be used to explicitly close the DB connection and return it to the pool. This operation doesn’t have any response."
+metadata:
+  - name: url
+    required: true
+    description: "Represent DB connection in Data Source Name (DNS) format. See here SSL details"
```
Comment (ItalyPaleAle) [inline bindings/mysql/metadata.yaml:24]:
```suggestion
    description: "Represents a DB connection in Data Source Name (DSN) format"
    url:
      title: "Enabling SSL"
      url: "https://docs.dapr.io/reference/components-reference/supported-bindings/mysql/#ssl-connection"
```

File: bindings/mysql/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    example: "user:password@tcp(localhost:3306)/dbname"
+    type: string
+  - name: pemPath
+    required: true
```
Comment (ItalyPaleAle) [inline bindings/mysql/metadata.yaml:28]: This is not required (our docs are wrong)

File: bindings/mysql/metadata.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    example: "10"
+    type: number
+  - name: connMaxLifetime
+    required: false
+    description: "The max connection lifetime. Duration string"
+    example: "12s"
+    type: duration
+  - name: connMaxIdleTime
+    required: false
+    description: "The max connection idel time. Duration string"
```
Comment (ItalyPaleAle) [inline bindings/mysql/metadata.yaml:49]:
```suggestion
    description: "The max connection idle time."
```

File: bindings/mysql/metadata.yaml [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+    example: "10"
+    type: number
+  - name: maxOpenConns
+    required: false
+    description: "The max open connections. Integer greater than 0"
+    example: "10"
+    type: number
+  - name: connMaxLifetime
+    required: false
+    description: "The max connection lifetime. Duration string"
```
Comment (ItalyPaleAle) [inline bindings/mysql/metadata.yaml:44]:
```suggestion
    description: "The max connection lifetime."
```

**Comment by MregXN** [comment]
I have correct them in tha latest commit. Thanks for your comment. @ItalyPaleAle

**Pull request merged**